Console/Command/Convert.php

```diff
@@ -12,6 +12,9 @@
 use Magento\CloudPatches\Patch\Status\StatusPool;
 use Magento\Framework\App\Filesystem\DirectoryList;
 use Magento\Framework\App\ProductMetadata;
```

Comment:
If the implementation is replaced with the following Interface `\Magento\Framework\App\ProductMetadataInterface`, then this code will support Commerce edition, as the current code always returns "Community" even when ran in a "Commerce" edition.